websurfx/config.lua

```diff
@@ -20,7 +20,6 @@ theme = "simple" -- the theme name which should be used for the website
 -- Caching
 redis_connection_url = "redis://127.0.0.1:8082" -- redis connection url address on which the client should connect on.
 
--- Aggregator
-aggregator = {
-  random_delay = false, -- whether to add random delay before sending the request to the search engine
-}
\ No newline at end of file
+production_use = false -- whether to use production mode or not
+-- if production_use is set to true
+  -- there will be a random delay before sending the request to the search engines, this is to prevent the search engines from blocking the ip address
```

Comment:
```suggestion
production_use = false -- whether to use production mode or not (in other words this option should be used if it is to be used to host it on the server to provide a service to a large number of users)
-- if production_use is set to true
  -- There will be a random delay before sending the request to the search engines, this is to prevent DDoSing the upstream search engines from a large number of simultaneous requests.
```

I think mentioning `ip-blocking` will off-put many people from using the project and scare them instead of mentioning `DDoS` I feel would be more reasonable, also I think production should have some explanation as this makes it clear as to why that option has been added :smile: .